mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -107,6 +107,11 @@
       "run": "after",
       "snippetPath": "remove_deprecated_data.sql",
       "fromModuleVersion": "mod-source-record-manager-2.3.1"
+    },
+    {
+      "run": "after",
+      "snippet": "ALTER TABLE journal_records ADD COLUMN IF NOT EXISTS title text NULL;",
+      "fromModuleVersion": "mod-source-record-manager-2.5.0-SNAPSHOT"
```

Comment:
please remove SNAPSHOT from the version here

Reply:
Yes. Thank you @KaterynaSenchenko! I forget to remove it after local testing.
Done.